jwst/white_light/white_light.py

```diff
@@ -40,9 +40,20 @@ def white_light(input):
     log.debug("norders = %d, sporders = %s, ntables_order = %s",
               norders, str(sporders), str(ntables_order))
 
-    # Compute the flux sum for each integration in the input
+    # Create a wavelength mask, using cutoffs if specified, then
+    # compute the flux sum for each integration in the input.
+    low_cutoff = -1.
```

Comment:
I'm guessing 0 (zero) would've been an appropriately small default lower-limit, but going negative can't hurt (just in case someone observes something with a VERY high blue shift)